Project_Json/ArgumentJson.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Project_Interface;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Project_Json
+{
+    public class ArgumentJson : IFileHandling
+    {
+        string[] _clArguments = null;
+
+        public ArgumentJson(string[] clArguments)
```

Comment:
`Scope`: This is a class that parses data.  Should it really need to know about command line arguments?